lib/gringotts/gateways/pin_pay.ex

```diff
@@ -73,7 +73,7 @@ defmodule Gringotts.Gateways.PinPayments do
   - To save you time, we recommend [cloning our example
   repo][example] that gives you a pre-configured sample app ready-to-go.
   + You could use the same config or update it the with your "secrets"
-          as described [above](#module-registering-your-monei-account-at-PinPay).
+          as described [above](#module-configuring-your-pinpay-account-at-gringotts).
```

Comment:
This is a hyperlink and you cannot set it arbitrarily. Please make it as I have suggested, it will not work otherwise. You can check that it works by building the docs using `mix docs` and clicking on it.